plugins/signalmonitor/signalhistoryview.cpp

```diff
@@ -58,11 +58,11 @@ SignalHistoryView::SignalHistoryView(QWidget *parent)
     connect(m_eventDelegate, SIGNAL(visibleIntervalChanged(qint64)),
             this, SIGNAL(delegateVisibleIntervalChanged(qint64)));
     connect(m_eventDelegate, SIGNAL(visibleOffsetChanged(qint64)),
-            this, SLOT(eventDelegateChanged()));
+            this, SLOT(eventDelegateChanged()), Qt::QueuedConnection);
```

Comment:
Qt does _not_ group similar events, so this whole change set is highly dubious to me. If you want to group stuff, you'll have to implement the grouping yourself.